addons/crm/wizard/crm_lead_to_opportunity.py

```diff
@@ -41,20 +42,28 @@ def default_get(self, fields):
                 result['team_id'] = lead.team_id.id
             if not partner_id and not lead.contact_name:
                 result['action'] = 'nothing'
+
         return result
 
     name = fields.Selection([
         ('convert', 'Convert to opportunity'),
         ('merge', 'Merge with existing opportunities')
     ], 'Conversion Action', required=True)
+    action = fields.Selection([
+        ('create', 'Create a new customer'),
+        ('exist', 'Link to an existing customer'),
+        ('nothing', 'Do not link to a customer')
```

Comment:
This is added in partner.mass wizard. We should remove the addition in that wizard

Reply:
Done